administration/intro.adoc

```diff
@@ -103,42 +103,43 @@ Once Kubernetes is deployed, you will need to deploy the KubeVirt
 add-on. There are 2 ways to deploy the add-on to a cluster using the `kubectl` tool and
 manifest files:
 
-1. Stable flow
+1. Primary flow
 +++++++++++++++
 
-The stable flow installs KubeVirt using a manifest file, which contains all deployments and configurations of the
-KubeVirt core components:
+The primary flow installs KubeVirt using a operator, which manages lifecycle of all KubeVirt core components:
 
 [source,bash]
 ----
-$ RELEASE=v0.14.0
-$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt.yaml
+$ RELEASE=v0.15.0
+# creates KubeVirt operator
+$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt-operator.yaml
+# creates KubeVirt KV custom resource
+$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt-cr.yaml
 ----
 
-2. Alternative flow (aka Operator flow)
-+++++++++++++++++++++++++++++++++++++++
-
-Since version 0.13.0 KubeVirt can be installed using the new KubeVirt Operator and a KubeVirt custom resource.
-
-_________________________________________________________________________
-Note: The Operator is considered to be experimental for the time being.
-There is currently no way to upgrade KubeVirt using this flow.
-_________________________________________________________________________
+Since KubeVirt version 0.14.0 and k8s version 1.11.* you can also use `kubectl` for waiting until all KubeVirt components are "Ready".
 
 [source,bash]
 ----
-$ export VERSION=v0.14.0
-$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/$VERSION/kubevirt-operator.yaml
-$ kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/$VERSION/kubevirt-cr.yaml
+$ kubectl wait kv kubevirt --for condition=Ready
 ----
 
-Since version 0.14.0 you can also use `kubectl` for waiting until all KubeVirt components are "Ready".
+2. Alternative flow
```

Comment:
I would call it obsolete and retire it completely in a release or two if possible.

Reply:
done